### PR TITLE
seismology: Add introduction to origin/location/magnitude

### PR DESCRIPTION
**Pull request by core-man**
Address #17.

### CODE REVIEW

**Comment by core-man** [comment]
Only a little introduction to the seismic source is added. I'd like to work on it with additional PRs later.

File: source/seismology/seismic-source.rst [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+   本小节尚未开始编写。
+
+地震三要素
+----------
+
+对于地震，人们最关心的基本参数有三个（俗称\ **地震三要素**\），即时间、地点、强度（简称\ **时空强**\ ）：
+
+- 时间：地震发生时间（origin time），常采用协调世界时（即 Coordinated Universal Time，简写 UTC）
+- 地点：地震发生位置（hypocenter），其向上垂直投影到地面的位置称为震中（epicenter）
+- 强度：地震的大小（magnitude）
```
Comment (seisman) [inline source/seismology/seismic-source.rst:28]: Should we mention "震级" here?

File: source/seismology/seismic-source.rst [hunk 2]
```diff
@@ -1,2 +1,64 @@
...
+   :width: 70%
+   :align: center
+
+   1964-2001 年全球 5.2 级以上地震的年发生率和震源深度的关系。
+   引自《\ `Deep Earthquakes <https://doi.org/10.1017/CBO9781107297562>`__\ 》图 4.3。
+
+震级越大的地震，数目越少。震级增加一级，地震释放的能量增加 32 倍。
+一般 4 级以下的地震破坏能力较小，一些重要的全球地震目录也只收录 5 级以上的地震。
+
+.. figure:: earthquake-energe.jpg
```
Comment (seisman) [inline source/seismology/seismic-source.rst:54]:
```suggestion
.. figure:: earthquake-energy.jpg
```

File: source/seismology/seismic-source.rst [hunk 3]
```diff
@@ -1,2 +1,63 @@
...
+   本小节尚未开始编写。
+
+地震三要素
+----------
+
+对于地震，人们最关心的基本参数有三个（俗称\ **地震三要素**\），即时间、地点、强度（简称\ **时空强**\ ）：
+
+- 时间：地震发生时间（origin time），常采用协调世界时（即 Coordinated Universal Time，简写 UTC）
+- 地点：地震发生位置（hypocenter），其向上垂直投影到地面的位置称为震中（epicenter）
+- 强度：地震的大小（magnitude）
```
Comment (core-man) [inline source/seismology/seismic-source.rst:28]:
```suggestion
- 强度：地震的大小，常用震级（magnitude）表示
```

Comment (seisman) [reply source/seismology/seismic-source.rst:28]: Looks good.

**Review by seisman — APPROVED** [review]

**Pull request merged**